pages/screening_subject_search/contact_with_patient_page.py

```diff
@@ -0,0 +1,46 @@
+from playwright.sync_api import Page, expect, Locator
+from pages.base_page import BasePage
+from enum import Enum
+from utils.calendar_picker import CalendarPicker
+
+
+class ContactWithPatientPage(BasePage):
+    def __init__(self, page: Page):
+        super().__init__(page)
+        self.page = page
+
+        # Contact With Patient - Page Locators
+        self.contact_direction_dropdown = page.locator("#UI_DIRECTION")
+        self.contact_made_between_Patient_and_dropdown = page.locator("#UI_CALLER_ID")
```

Comment:
make Patient all lower case:
```suggestion
        self.contact_made_between_patient_and_dropdown = page.locator("#UI_CALLER_ID")
```

Reply:
Hi Adriano, As per your comment, Code refactoring is done & changes are implemented. Kindly review it now. Thanks!